crates/subspace-networking/src/create.rs

```diff
@@ -144,7 +144,6 @@ pub enum RelayConfiguration {
 
     /// Relay-client configuration in requesting mode (a client for a server behind NAT).
     /// It will enable creating a relay circuit.
-    /// Expected format for Multiaddr: /ip4/127.0.0.1/tcp/50000/p2p/<server_peer_id>/p2p-circuit
```

Comment:
This last commit is exactly the kind of change I'd squash with corresponding earlier commit before pushing (or force-pushed if already submitted), it doesn't really deserve its own commit.

Reply:
I agree. However, I found it in the last moments on the PR creation (self-review) after all the previous squashes. I'll force-push the branch next time.